### PR TITLE
docs: updated manual for bigquery

### PR DESCRIPTION
**Pull request by adam-stasiak**
### SUMMARY
I added information about required dependencies for running BigQuery with docker + info about required permissions for this.

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: docs/src/pages/docs/Connecting to Databases/google-bigquery.mdx [hunk 1]
```diff
@@ -9,7 +9,19 @@ version: 1
...
+```flask-cors
+vertica-python
+fbprophet
+sqlalchemy-vertica-python
+redis
+pybigquery
+pandas-gbq
+pydruid```
+
+With this modification all required dependencies will be installed with docker-compose up command.
```
Comment (villebro) [inline docs/src/pages/docs/Connecting to Databases/google-bigquery.mdx:24]: As this is a generic requirement for adding dependencies to `docker-compose`, I think this should not be under the BigQuery section (some of these deps are not specific to BQ).

**Review by villebro — COMMENTED** [review]
The permission addition is good, but I'd probably try to make the docker instruction more generic.

File: docs/src/pages/docs/Connecting to Databases/google-bigquery.mdx (hunk 1 above)
Comment (srinify) [reply docs/src/pages/docs/Connecting to Databases/google-bigquery.mdx:24]: Just seeing this now, I agree with @villebro that this should probably live in this section instead:

https://superset.apache.org/docs/databases/installing-database-drivers

@adam-stasiak let me know if you need any help or want to discuss this further

File: docs/src/pages/docs/Connecting to Databases/google-bigquery.mdx [hunk 2]
```diff
@@ -9,30 +9,20 @@ version: 1
...
-```
-
-When adding a new BigQuery connection in Superset, you'll also need to add the GCP Service Account
+When adding a new BigQuery connection in Superset, you'll need to create and use GCP Service Account
 credentials file (as a JSON).
 
 1. Create your Service Account via the Google Cloud Platform control panel, provide it access to the
    appropriate BigQuery datasets, and download the JSON configuration file for the service account.
 
-2. n Superset, Add a JSON blob to the **Secure Extra** field in the database configuration form with
```
Comment (mvoitko) [inline docs/src/pages/docs/Connecting to Databases/google-bigquery.mdx:26]: It's unclear how to provide service account for Superset.

File: docs/src/pages/docs/Connecting to Databases/google-bigquery.mdx [hunk 3]
```diff
@@ -9,30 +9,20 @@ version: 1
 ## Google BigQuery
 
 The recommended connector library for BigQuery is
-[pybigquery](https://github.com/mxmzdlv/pybigquery).
+[pybigquery](https://github.com/mxmzdlv/pybigquery). 
 
-The connection string for BigQuery looks like:
+In order to use pybigquery with docker-compose it is needed to extend requirements file with BigQuery specific packages described in [installing-database-drivers guide](../installing-database-drivers).
```
Comment (mvoitko) [inline docs/src/pages/docs/Connecting to Databases/google-bigquery.mdx:14]: It would be nice also to provide instructions for SuperSet deployed in k8s

**Comment by amitmiran137** [comment]
closing due to inactivity

**Pull request closed without merging**